src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -69,6 +69,24 @@ public override BinaryAnalyzerContext InitializeGlobalContextFromOptions(Analyze
 
             // Update context object based on command-line parameters.
             context.SymbolPath = options.SymbolsPath;
+
+            IList<string> inaccessibleSymbolPaths = DriverUtilities.GetInaccessibleSymbolPaths(context.SymbolPath);
+
+            if (inaccessibleSymbolPaths.Count > 0)
+            {
+                // These symbol servers provided are inaccessible: {0}.
+                context.Logger.LogConfigurationNotification(
+                    Errors.CreateNotification(
+                        context.CurrentTarget?.Uri,
+                        "OneOrMoreSymbolServersProvidedInaccessible",
+                        ruleId: null,
+                        FailureLevel.Warning,
+                        exception: null,
+                        persistExceptionStack: false,
+                        messageFormat: RuleResources.OneOrMoreSymbolServersProvidedInaccessible,
```

Comment:
>OneOrMoreSymbolServersProvidedInaccessible [](http://example.com/codeflow?start=53&length=42)

OneOrMoreSymbolServersAreNotAccessible.

When you create user-facing strings, please provide a real example as surfaced by the tool.